src/Components/RelatedProducts/RPComponents/ProductCard/ProductCard.jsx

```diff
@@ -20,65 +22,41 @@ const ProductCard = (props) => {
   if (card.id === undefined) {
     return (<EmptyCard />);
   }
-  if (props.carouselType === 'suggestions') {
-    return (
-      <div className="product-card-main">
-        <div className="container">
-          <div className="row product-card-img-top">
-            <ProductCardImage cardProduct={card.id} />
-          </div>
-          <div className="row product-card-info-bottom">
-            <div className="container-fluid product-card-body">
-              <div className="row product-card-category">
-                {card.category}
-              </div>
-              <div className="row product-card-title">
-                {card.name}
-              </div>
-              <div className="row product-card-price">
-                $
-                {card.default_price}
-                .00
-              </div>
-              <div className="row product-card-star">
-                <Stars />
-              </div>
-            </div>
-          </div>
+
+  return (
```

Comment:
This Is All Code CleanUp

Reply:
A Conditional Render Was Removed In Favor Of A Dynamic Child Determined By Options Passed As Props